README.md

```diff
@@ -1,24 +1,249 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
```

Comment:
### I'm just using the readme file to write comments because not all files are included in this PR you should modify the code that is responsible for what is shown below 

- One of the requirements is to follow the given design by implementing the same colors, font-family, spaces, and layout you can see the screenshot of the requirements below 
<img width="546" alt="Screenshot 2023-02-15 at 15 24 42" src="https://user-images.githubusercontent.com/61790587/219056526-1c6f88ad-759e-45a9-805d-f8aa0f2e4ded.png">

- [ ] it would be better to stick to the design make the login page the same as in the design by adding the same font size, and spaces, and layout 

- Login page design 
<img width="238" alt="Screenshot 2023-02-15 at 15 27 10" src="https://user-images.githubusercontent.com/61790587/219057655-c2387507-4103-4f64-8361-98dc79bdb09d.png">
- Yours 

![Screenshot 2023-02-15 at 15 27 39](https://user-images.githubusercontent.com/61790587/219057938-368f1164-0e8b-48e8-a27d-ce9e62b5dbe9.png)

- Font family from the Design 
<img width="412" alt="Screenshot 2023-02-15 at 15 25 53" src="https://user-images.githubusercontent.com/61790587/219057802-99df8d3a-3e32-4603-badf-f2e71e8bd4c0.png">

Reply:
- [ ] @NatanimA great work implementing the changes requested however the design needs some changes to match the design given, the input area length and also the button placement below is a comparison,


Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/219132982-7e94a1d6-acf5-4723-9001-52a921d22bc0.png)  |  ![image](https://user-images.githubusercontent.com/61408860/219133036-4ab4f44d-f791-417b-9e1e-3bd0e255c575.png)